MusicTaste/MusicTaste/MatchingAlgorithm.m

```diff
@@ -16,38 +16,34 @@ + (instancetype)shared {
     return sharedManager;
 }
 
-+ (void) lookForMatches{
+- (void) lookForMatches{
     NSMutableDictionary *matchDict = [NSMutableDictionary new];
     PFObject *music = [PFObject objectWithClassName:@"Music"];
     PFQuery *query = [PFQuery queryWithClassName:@"Music"];
     PFUser *current = [PFUser currentUser];
     //Goes through Query
     [query findObjectsInBackgroundWithBlock:^(NSArray *spotifyData, NSError *error) {
         for (PFObject *user in spotifyData){
-            //We make sure we are not comparing the same object
-            if (![user.objectId isEqual:music.objectId]){
```

Comment:
Was unnecessary to check if ids are equal here when I can do that in comparison function